aggregator/src/multicast.rs

```diff
@@ -1,64 +0,0 @@
-//! A set of abstractions for dealing with `ReliableMulticast` in a more testable
```

Comment:
The multicast trait was an unnecessary wrapper on `rmc` with minimal own logic.
After the `rmc` interface update it is no longer compatible for wrapping `rmc`. Decided to get rid of it.

Reply:
I don't know `Multicast`'s original purpose. `rmc` changed so much that I'm not sure whether adjusting `Multicast` to `rmc` so that the interface of `Multicast` is preserved is feasible. It was easier for me to handle new `rmc` in `aggregator` directly. Overall this code has gotten much simpler so idk. Feel like adjusting `Multicast` for the new `rmc` would be lying to ourserves and not using the simplicity of the new `rmc`. Even `aggregator` could be simplified but I didn't want to change too much.